store/migrations/0046_auto_20221212_1627.py

```diff
@@ -1,23 +0,0 @@
-# Generated by Django 3.2.8 on 2022-12-12 21:27
-
-from django.db import migrations, models
-
-
-class Migration(migrations.Migration):
-
-    dependencies = [
-        ('store', '0045_auto_20221103_1538'),
-    ]
-
-    operations = [
-        migrations.AddField(
-            model_name='historicalorder',
-            name='is_made_by_admin',
-            field=models.BooleanField(default=False, verbose_name='Is made by admin'),
-        ),
-        migrations.AddField(
-            model_name='order',
-            name='is_made_by_admin',
-            field=models.BooleanField(default=False, verbose_name='Is made by admin'),
```

Comment:
Deleting migration is not a good idea!

Reply:
I don't understand what happened, the file got doubled with just the hour changing, the content is the same, I don't know how to solve this clean so we'll check tomorrow